### PR TITLE
Fixed some typos in the lesson

### PR DESCRIPTION
**Pull request by helloitsm3**
Fixed typo in section 3 lesson 1 of Solana Metaplex lesson
   - Changed "add the follow code" to "add the following code"

Fixed typo in Solana Metaplex lesson
  - Reduced Solana airdrop amount from 5 to 2.

### CODE REVIEW

**Comment by adilanchian** [comment]
Hey @helloitsm3 !

Before we continue here, can you do one more fetch from the remote to make sure you are up to date :)?

**Comment by helloitsm3** [comment]
> re

Yup, done that :)

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md [hunk 1]
```diff
@@ -61,27 +56,9 @@ const instructions = [
...
-    TOKEN_PROGRAM_ID,
-    mint.publicKey,
-    token,
-    walletAddress.publicKey,
-    [],
-    1
-  ),
+  Token.createInitMintInstruction(TOKEN_PROGRAM_ID, mint.publicKey, 0, walletAddress.publicKey, walletAddress.publicKey),
+  createAssociatedTokenAccountInstruction(token, walletAddress.publicKey, walletAddress.publicKey, mint.publicKey),
+  Token.createMintToInstruction(TOKEN_PROGRAM_ID, mint.publicKey, token, walletAddress.publicKey, [], 1),
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:61]: Can you revert this please?

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md [hunk 2]
```diff
@@ -106,17 +83,17 @@ This you already know! We set up a provider, then call `mintNft` which is a func
...
       const { result } = notification;
       if (!result.err) {
-        console.log('NFT Minted!');
+        console.log("NFT Minted!");
         await getCandyMachineState();
       }
     }
   },
-  { commitment: 'processed' }
+  { commitment: "processed" }
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:96]: You can revert this as well :). We like single quotes hehe

**Review by adilanchian — CHANGES_REQUESTED** [review]
Looking good! Just left a few comments for you :)

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md (hunk 2 above)
Comment (helloitsm3) [reply Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md]: Done :)

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .DS_Store?
+.vscode
```
Comment (adilanchian) [inline .gitignore:3]: We actually can remove this!  :)

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md [hunk 4]
```diff
@@ -282,11 +269,11 @@ Nice. Now that we have this function ready to call, we actually need to place it
...
   machineStats && (
-    <div className="machine-container">
+    <div className='machine-container'>
       <p>{`Drop Date: ${machineStats.goLiveDateTimeString}`}</p>
       <p>{`Items Minted: ${machineStats.itemsRedeemed} / ${machineStats.itemsAvailable}`}</p>
-      <button className="cta-button mint-button" onClick={mintToken}>
-          Mint NFT
+      <button className='cta-button mint-button' onClick={mintToken}>
+        Mint NFT
       </button>
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:277]: Can we keep this double quotes please?

**Review by adilanchian — CHANGES_REQUESTED** [review]
Left some thoughts :) ty!

File: Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md [hunk 5]
```diff
@@ -61,7 +56,7 @@ const instructions = [
     lamports: rent,
     programId: TOKEN_PROGRAM_ID,
   }),
-  Token.createInitMintInstruction(
+   Token.createInitMintInstruction(
```
Comment (adilanchian) [inline Solana_NFTs/en/Section_3/Lesson_2_Create_Mint_NFT_Button.md:59]: This space should be removed!

**Review by adilanchian — CHANGES_REQUESTED** [review]
Left another small comment for you !

Also - can you please not commit your `.vscode` file? 

Thanks!

**Comment by NickGonzalez04** [comment]
@adilanchian This looks good to go! Will come back around and make sure this is still good since the Candy machine went to V2 and there were some dock changes.

**Comment by adilanchian** [comment]
Hey @NickGonzalez04 sounds good!! ty :)

**Comment by adilanchian** [comment]
Closing do to inactivity !

**Pull request closed without merging**